src/js/Draw/L.PM.Draw.Line.js

```diff
@@ -291,6 +291,8 @@ Draw.Line = Draw.extend({
     // sync the hintline again
     this._syncHintLine();
     this._setTooltipText();
+
+    this._fireVertexRemoved(marker, undefined, 'Draw');
```

Comment:
```suggestion
    this._fireVertexRemoved(marker, undefined, 'Draw');
```

Let's check if we could/should send the old indexPath or an empty array instead of undefined

Reply:
To keep my first ideas
```    
const markers = this._layerGroup.getLayers();
    // the index path to the marker inside the multidimensional marker array
    const { indexPath } = .findDeepMarkerIndex(
      markers,
      marker
    );
```